scripts/footprint/plan.txt

```diff
@@ -10,3 +10,7 @@ footprints,default,ehl_crb,tests/benchmarks/footprints,
 footprints,userspace,ehl_crb,tests/benchmarks/footprints,-DCONF_FILE=prj_userspace.conf
 echo_client,default,frdm_k64f,samples/net/sockets/echo_client,
 echo_server,default,frdm_k64f,samples/net/sockets/echo_server,
+bt_beacon, default, nrf52840dk_nrf52840, samples/bluetooth/beacon
+bt_peripheral, default, nrf52840dk_nrf52840, samples/bluetooth/peripheral
+bt_central, default, nrf52840dk_nrf52840, samples/bluetooth/central
```

Comment:
The central sample does almost nothing, but the central_hr sample is a GATT client doing GATT discovery. I think that might be a better sample for this.

Reply:
```suggestion
bt_central_hr, default, nrf52840dk_nrf52840, samples/bluetooth/central_hr
```